control-plane/agents/src/bin/core/node/service.rs

```diff
@@ -119,6 +119,32 @@ impl NodeOperations for Service {
         let node = self.drain(id, label).await?;
         Ok(node)
     }
+
+    /// Apply the label to node.
+    async fn label(
+        &self,
+        id: NodeId,
+        label: Option<HashMap<String, String>>,
+    ) -> Result<Node, ReplyError> {
+        let label_map = label.ok_or_else(|| SvcError::InvalidLabel {
+            labels: "".to_string(),
+            resource_kind: ResourceKind::Node,
+        })?;
+        let node = self.label(id, label_map).await?;
+        Ok(node)
+    }
+    /// Remove the specified label from the node.
+    async fn unlabel(&self, id: NodeId, label: String) -> Result<Node, ReplyError> {
```

Comment:
What is label here, is it a specific key=val, or just key, or multiple?

Reply:
It is just a key ,
For the command : `kubectl mayastor label node A-`. A will be passed to the `unlabel` function. And this will remove the key from the label of the node